libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/SpeechListener.java

```diff
@@ -1,10 +1,10 @@
 package com.mapbox.services.android.navigation.ui.v5.voice;
 
-interface InstructionListener {
+interface SpeechListener {
 
   void onStart();
 
   void onDone();
 
-  void onError(boolean isMapboxPlayer, String errorText);
+  void onError(String errorText, SpeechAnnouncement speechAnnouncement);
```

Comment:
Question - Could we refactor the code so we don't need to pass the announcement in the error callback?

Reply:
Let's stick with the current setup here - this is an internal listener.  No excuse, though, I know.  We can look into re-wiring in the future.